docarray/doc_index/abstract_doc_index.py

```diff
@@ -85,10 +86,13 @@ def __init__(self, db_config=None, **kwargs):
                 'A DocumentIndex must be typed with a Document type.'
                 'To do so, use the syntax: DocumentIndex[DocumentType]'
             )
+        self._logger = logging.getLogger('docarray')
         self._db_config = db_config or self.DBConfig(**kwargs)
         if not isinstance(self._db_config, self.DBConfig):
             raise ValueError(f'db_config must be of type {self.DBConfig}')
+        self._logger.info('DB config created')
```

Comment:
isn't info too much intrusive ? I would say 'debug' by default

Reply:
Agree with Jack.

I was following python's [tutorial about logging](https://docs.python.org/3/howto/logging.html), and based on the definition on what should be logged as info and debug (DEBUG | Detailed information, typically of interest only when diagnosing problems. INFO | Confirmation that things are working as expected.) I think we should either keep it as INFO or remove the log (in case it doesn't bring much value) 

I'd say let's keep it, and if the tests are too verbose we can leave the log level to default (WARNING)

Reply:
Default logging level is WARNING, so you won't see this by default, if that's your concern here.
I set logging level to INFO for tests only, so you'll see this when running tests.

Reply:
Ah okay then makes sense !